orc8r/cloud/go/obsidian/swagger/spec_servicer.go

```diff
@@ -30,6 +31,7 @@ type specServicer struct {
 // NewSpecServicerFromFile intializes a spec servicer
 // given a service name.
 func NewSpecServicerFromFile(service string) protos.SwaggerSpecServer {
+	service = strings.ToLower(service)
```

Comment:
:+1:

For my understanding: did you run into an issue or did you just read how we do this in other areas of the codebase?

Reply:
When I was checking the runtime logs I noticed that services that were in all CAPS(e.g. CTRACED) were unable to register so I dig some digging and found this to be the best solution.